File: test/fixtures/sample.java
public class ShoppingCart {

  public double calculateTotal(double[] prices) {
    double sum = 0;
    for (double price : prices) {
      sum += price;
    }
    return sum;
  }

  public double applyToAll(double[] prices) {
    return calculateTotal(prices);
  }
}

File: test/fixtures/sample.py
def compute_discount(price, percent):
    """Compute the discounted price."""
    return price * (1 - percent / 100)


def compute_discount_for(prices, percent):
    return [compute_discount(p, percent) for p in prices]


def apply_discount(price, percent):
    return compute_discount(price, percent)